src/musicbrainz/tagfetcher.cpp

```diff
@@ -27,6 +27,9 @@
 #include <QUrl>
 #include <QtConcurrentMap>
 
+const std::array<QString, 5> TagFetcher::kFetchedFields{
```

Comment:
Not sure, `Song::kColumns` is a similar id list; could merge them but I think there might be fields in `Song` that aren't necessarily in the database.